jax/_src/custom_derivatives.py

```diff
@@ -564,21 +565,17 @@ def __call__(self, *args: Any, **kwargs: Any) -> ReturnValue:  # pytype: disable
       out_tree = aux if fst else aux[0]
       return tree_unflatten(out_tree, out_flat)
 
-@partial(partial, tree_map)
 def _check_for_tracers(x):
-  if isinstance(x, core.Tracer):
-    msg = ("Found a JAX Tracer object passed as an argument to a custom_vjp "
-           "function in a position indicated by nondiff_argnums as "
-           "non-differentiable. Tracers cannot be passed as non-differentiable "
-           "arguments to custom_vjp functions; instead, nondiff_argnums should "
-           "only be used for arguments that can't be or contain JAX tracers, "
-           "e.g. function-valued arguments. In particular, array-valued "
-           "arguments should typically not be indicated as nondiff_argnums. "
-           "\n\n"
-           "This behavior recently changed in JAX. "
-           "See https://github.com/google/jax/blob/main/docs/custom_vjp_update.md "
-           "for more information.")
-    raise UnexpectedTracerError(msg)
+  for leaf in tree_leaves(x):
+    if isinstance(x, core.Tracer):
```

Comment:
```suggestion
    if isinstance(leaf, core.Tracer):
```
Did we mean to write this instead here?

Reply:
Yes!

Reply:
fixed in https://github.com/google/jax/commit/c8a7d5990daecdbf3ff5a25d67ad4979a55e2182